### DISCUSSION TITLE
A Simple Write-First Dual Port Ram

### DISCUSSION THREAD

**Discussion by trayres** [logisim-evolution/logisim-evolution#838] [Show and tell]
![image](https://user-images.githubusercontent.com/168285/126602334-2b8514b6-99e1-4f4f-9a6c-bcda1afb073c.png)
I created a simple dual port memory by emulating the behavior using multiple cycles. While this is inefficient, it might come in handy for certain kinds of retro-computing simulations (e.g. the RAM of an AM2901, which I believe this would work for).

This is a "Write-First" memory - that is, if address 0 is written to and then read from, the written value is read out. 
Included is a small test bench; please let me know if you find it useful (and especially let me know if you find a mistake).

The .circ is located here:
https://github.com/trayres/logisim-evolution-examples/blob/main/sdp_export.circ

**Comment by MarcinOrlowski** [comment]
One more note: you may want to zip your `*.circ` file and attach directly.
